class5.html

```diff
@@ -83,548 +68,158 @@ <h2>Using the Inspector</h2>
+              .article {
+                background-color: white;
               }
             </code>
           </pre>
-
-          <aside class="notes">
-            Header section complete!
-          </aside>
         </section>
 
         <section>
-          <h2>Add an Image</h2>
-          <p>
-            To add an image, we'll use the background-image CSS property but
-            we'll need some HTML so the CSS has HTML to target.
-          </p>
-
-          <pre>
-            <code class="html" data-trim contenteditable>
-              <div class="hero"></div>
-            </code>
-          </pre>
-        </section>
-
-        <section>
-          <h2>Select an Image</h2>
-          <a href="final_project_images.html" target="_blank">Image Options</a>
-        </section>
-
-        <section>
-          <h2>Add an Image</h2>
+          <h2>Style the Article (css)</h2>
+          <p>Add whitespace around the article text.</p>
 
           <pre>
             <code class="css" data-trim contenteditable>
-              .hero {
-                background-image: url("watermelon.jpg");
-                background-position: center;
-                background-repeat: no-repeat;
-                background-size: cover;
-                border-top: 1px solid #d6d6d6;
-                height: 700px;
+              .article {
+                background-color: white;
+                padding: 20px;
               }
             </code>
           </pre>
         </section>
 
         <section>
-          <h2>Create a Blog Post</h2>
-          <p>
-            Use Lorem Ipsum, also known as "dummy text", to simulate content.
-            Visit <a href="https://www.lipsum.com">https://www.lipsum.com</a> to
-            generate Lorem Ipsum.
-          </p>
-
-          <pre>
-            <code class="html" data-trim contenteditable>
-              <p>Best Florida Beaches</p>
-              <p>June 1, 2017</p>
-              <hr>
-              <p>Generated Lorem Ipsum...</p>
-            </code>
-          </pre>
-        </section>
-
-        <section>
-          <h2>Add a Link to the Blog Post</h2>
-
-          <pre>
-            <code class="html" data-trim contenteditable>
-              <p>Best Florida Beaches</p>
-              <p>June 1, 2017</p>
-              <hr>
-              <p>Generated Lorem Ipsum...</p>
-              <p>
-                Read more about Florida Beaches
-                <a href="#">here</a>.
-              </p>
-            </code>
-          </pre>
-        </section>
-
-        <section>
-          <h2>Style the Blog Post Title</h2>
-          <p>Add a class to the blog post title.</p>
+          <h2>Style the Article Title (html & css)</h2>
+          <p>Add a class to the article title.</p>
+          <p>Style the article title to be larger and bold.</p>
 
           <pre>
             <code class="html" data-trim contenteditable>
               <p class="article-title">Best Florida Beaches</p>
```

Comment:
```suggestion
              <h2 class="article-title">Best Florida Beaches</h2>
```

Potentially another good place for semantic elements.